### PR TITLE
Tests for 1.3.4 and 1.3.5

### LINKED ISSUE #99: 1.3.2,1.3.3, 1.3.4 Tests for Investments API
## Prerequisite

- [X] I am using the latest version of the code, and checked the documentation related to my needs.

* **I'm submitting a ... (remove the irrelevant one)**
  - [X] test request

* **Briefly describe the nature of the issue. If test request, please provide the functionality. If feature request, please provide related requirement.**

Tests for loss/profit calculation are needed. 

* **Deadline for resolution:** 
May 6, 2019

### PR DESCRIPTION
**Pull request by mertyg**
# Pull Request Template

## Description

Please include a summary of the work done and which issue is addressed.

Fixes #99

## Type of change

Please delete options that are not relevant.

- [X] Test

## How Has This Been Tested?

- [X] My code follows pep8
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have committed tests that prove my fix is effective or that my feature works. Or, I have opened an issue.
- [X] I have added 2 reviewers to get the approval of.

### CODE REVIEW

File: practice-app/backend/api/tests.py [hunk 1]
```diff
@@ -173,4 +173,48 @@ def test(self):
...
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        investment_id = response.json()["investments"][0]["id"]
+        response = self.client.post('/investments/profit',
+                                    data={"investment_id": investment_id, "symbol": "SYM1"})
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        profit = response.data["profit"]
+        self.assertEqual(profit, -133.33)
+        response = self.client.post('/investments/total_profit',
+                                    data={"investment_id": investment_id, "symbol": "SYM1"})
```
Comment (buseg) [inline practice-app/backend/api/tests.py]: I didn't understand why you needed to give investment_id as data here, isn't total_profit able to calculate the total profit of user by just getting user_id?

Comment (mertyg) [reply practice-app/backend/api/tests.py]: right fixing now

**Comment by buseg** [comment]
Added some new tests please check them.

File: practice-app/backend/api/tests.py [hunk 2]
```diff
@@ -173,4 +173,112 @@ def test(self):
...
+        self.assertTrue("token" in response.data)
+        token = response.data["token"]
+        token = "jwt " + token
+
+        self.client.credentials(HTTP_AUTHORIZATION=token)
+
+        response = self.client.post('/investments/total_profit',
+                                    data={"symbol": "SYM2"})
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(response.data["total_profit"], -630)
```
Comment (yemreinci) [inline practice-app/backend/api/tests.py]: Its better to use assertAlmostEqual() when comparing floating point numbers.

**Pull request merged**